### PR TITLE
Hide postal carrier image if no tracking url

### PR DESCRIPTION
**Pull request by exu3**
Currently, it shows an image of the logo of the postal carrier for a card on the way but the negative margin makes the image overflow. This PR hides the postal carrier logo unless `shipping_has_tracking`.
Screen Shot 2021-11-11 at 9 00 22 PM

now:
Screen Shot 2021-11-11 at 9 00 03 PM

### CODE REVIEW

File: app/views/stripe_cards/shipping.html.erb [hunk 1]
```diff
@@ -37,15 +37,15 @@
...
+      <% if card.shipping_has_tracking? %>
+      <footer class="flex flex-wrap items-center p2 relative">
         <%= image_tag "https://logo.clearbit.com/#{card.stripe_obj[:shipping][:carrier]}.com", width: 64, height: 64, style: 'height: auto; object-fit: contain;', class: 'rounded bg-white mr2' %>
-        <% if card.shipping_has_tracking? %>
           <%= link_to card.stripe_obj[:shipping][:tracking_url], class: 'btn' do %>
             <%= inline_icon 'external' %>
             Track shipment
           <% end %>
+        </footer>
         <% end %>
```
Comment (garyhtou) [inline app/views/stripe_cards/shipping.html.erb]: just one small thing, could you fix the indent here?

**Review by garyhtou — APPROVED** [review]
good work!!

File: app/views/stripe_cards/shipping.html.erb (hunk 1 above)
Comment (exu3) [reply app/views/stripe_cards/shipping.html.erb]: Think I fixed it, lmk if that works :D

**Review by garyhtou — APPROVED** [review]
good to go! 👏

**Pull request merged**